app/components/custom_list/custom_list_row.js

```diff
@@ -49,7 +49,9 @@ export default class CustomListRow extends React.PureComponent {
                             </View>
                         </View>
                     }
-                    {this.props.children}
+                    <View style={style.children}>
```

Comment:
what other component or list is being affected by this?

Reply:
Affected components/screens are the following and looks the same as before after this change.  See screenshots of screens on description.

CustomListRow
- ChannelListRow
    - CustomFlatList
        - MoreDirectMessages (screen)
    - CustomList
        - ChannelAddMembers (screen)
    - CustomSectionList
        - MoreDirectMessages (screen)
    - MenuActionSelector (screen)
    - MoreChannels (screen)
- OptionListRow
    - MenuActionSelector (screen)
- UserListRow
    - CustomFlatList (see above)
    - CustomSectionList (see above)
    - ChannelAddMembers (screen)
    - ChannelMembers (screen)
    - MenuActionSelector (screen)
    - MoreDirectMessages (screen)